Doc/whatsnew/3.7.rst

```diff
@@ -473,7 +474,7 @@ cgi
 ---
 
 :func:`~cgi.parse_multipart` returns the same results as
-:class:`~FieldStorage` : for non-file fields, the value associated to a key
+:class:`~FieldStorage` for non-file fields, the value associated to a key
 is a list of strings, not bytes.
```

Comment:
The original makes more sense to me, and seems to be correct. I understand Pierre’s change always uses FieldStorage, regardless of whether a field represents a file or not. Otherwise you would need a new sentence or semicolon to start “The value associated . . .”.

Reply:
Then the space before `:` should be removed. And the whole entry should be moved to the "Porting to Python 3.7" section, since this is rather a compatibility-breaking change than a new feature.